README.md

```diff
@@ -129,6 +129,18 @@ gauge.get({ room: 'kitchen' })
 # => 21.534
 ```
 
+Also you can use gauge as the bi-directional counter:
+
+```ruby
+gauge = Prometheus::Client::Gauge.new(:concurrent_requests_total, '...')
+
+gauge.increment({ service: 'foo' })
```

Comment:
Yes, the ruby client codebase is the oldest one and needs quite some rework. I think using increment/decrement for now makes the most sense to keep the codebase itself consistent. It'll all be updated with the rewrite.